AnnoyancesFilter/sections/popups.txt

```diff
@@ -2287,7 +2288,7 @@ com-magazin.de##w-div[role$="dialog"]
 ||muzofond.com/mithdl.js
 ||sud.ua/js/jquery.arcticmodal*.min.js
 ||turkusozu.com/js/fancybox/
-/jquery.magnific-popup.min.js$domain=ufatime.ru|koreanworld.ru
+/jquery.magnific-popup.min.js$domain=ufatime.ru|koreanworld.ru|abazero.com
```

Comment:
You are fixing antiadblock detection, so, move it to the `antiadblock` section.
Also there is a problem with page scrolling, you have to fix it.

Reply:
Is not it caused by FAB scriptlet?

Reply:
Yes, on my end it's the same. We can just remove this domain from the scriptlet part, it would be better.
But the problem with page scrolling is still here.

Reply:
@Sergey-Lyapin probably fab is not used here. In this case plz clean up the rules and make new.

Reply:
@zloyden I don't see scrolling problem (on an iPhone too), #wrapper contains all site's content and `overflow: visible` assigned to it.
Same without AdGuard, it is just a site's design.

Reply:
> I don't see scrolling problem

Because you added `abazero.com#$#html { overflow: visible!important; }` in next commit. Now it's ok.

Reply:
> Same without AdGuard (html with overflow-y: hidden), it is just a site's design.

No, I've checked without these rules and I've just forgotten to remove them